clients/admin-ui/src/features/common/PIIToggle.tsx

```diff
@@ -1,12 +1,21 @@
 import { Switch } from "@fidesui/react";
 import { setRevealPII } from "privacy-requests/privacy-requests.slice";
-import React, { ChangeEvent } from "react";
+import React, { ChangeEvent, useEffect } from "react";
 import { useDispatch } from "react-redux";
 
 const PIIToggle: React.FC = () => {
   const dispatch = useDispatch();
   const handleToggle = (event: ChangeEvent<HTMLInputElement>) =>
     dispatch(setRevealPII(event.target.checked));
+
+  useEffect(() => {
+    /*
+      PII should default to hidden on page load.
+      This ensures the state isn't reused as the user navigates around
+   */
+    dispatch(setRevealPII(false));
```

Comment:
if the PII toggle is indeed supposed to be independent from one another, perhaps we don't want to store this state in redux at all, and just use a regular ol' `useState` here?

reading the ticket, it seems like there are two issues:
1. This toggle wasn't showing on/off properly based off of the PII state. this is definitely a bug which could be fixed by setting an initial state based on a selector `selectRevealPII` from redux, though your change here also works because it just always sets it to false on load, ignoring whatever state there initially was
2. The toggle on the list page and the details page should be independent. It seems like this PII toggle was initially set up so that revealing PII would _not_ be independent, and that if you revealed once, then it's revealed everywhere (until you toggle again). This is def a product decision, and it seems like the decision was to make these totally independent, so if you want to unreveal PII you have to do it everywhere, but the code doesn't _quite_ reflect that. Again, your change definitely works since it always sets it to not reveal on load, but it makes things a little weird.

For example, we won't run into this exact use case since it'd be weird for two toggles to be on the same page, but if we had a page that had

```typescript
<PIIToggle id="one" />
<PIIToggle id="two" />

<PII data="name" />
```

And the user clicked `one`, then `<PII data="name" />` would reveal itself since it changes the global PII redux state. But `two` would stay untoggled since they're not linked. To me, the data model makes more sense as

```typescript
const [revealPII, setRevealPII] = useState(false)
<PIIToggle id="one" reveal={revealPII} onChange={setRevealPII} />
<PIIToggle id="two" reveal={revealPII} onChange={setRevealPII} />

<PII data="name" reveal={revealPII} />
```

In this way, it's clearer that it's not a global state that is being changed, but just the page's state, and all the components will respect that.

That involves changing the way `<PII />` works too though and also taking this state out of redux which is a bigger change. And the example I gave above is pretty contrived, so we likely won't run into it, but I can see it getting us in the future.

As an aside, are we sure that we actually want that second bullet point I listed above? I can see it getting annoying if people want to just keep PII revealed but they have to keep setting it. I know it's a product q, but if the answer is "no let's keep them linked", then we don't have any of the problems I just mentioned, and can just change the `<Switch />` component to properly take an initial state :)

Reply:
I completely agree that this is kind of jank solution and it probably shouldn't be in the redux state. Based off what the ticket is saying it seems like they're intended to be independent on each page. 

I did it this way because it was really quick but your proposed solution makes sense. I'll take a look real quick and see how much of a lift it would be to update it to work that way. If it's not too bad I'll update it to work that way in this PR.


Also, I think there may be another hidden bug included in #1. The persisted redux store that gets reloaded may have been setting the state causing the PII to be shown when the user visits the page again in the future. 

The current solution does fix that issue but switching over to a `useState` makes a lot more sense